tap_google_analytics/client.py

```diff
@@ -110,7 +110,12 @@ def _lookup_data_type(self, type, attribute, dimensions_ref, metrics_ref):
 
     @staticmethod
     def _generate_report_definition(report_def_raw):
-        report_definition = {"metrics": [], "dimensions": []}
+        report_definition = {
+            "metrics": [],
+            "dimensions": [],
+            "dimension_filters": [],
+            "metric_filters": [],
```

Comment:
@ajbosco would you mind adding a blurb in the README to explain that this feature is supported?